spec/pg_ha_migrations_spec.rb

```diff
@@ -5,6 +5,30 @@
     expect(ActiveRecord::Migration.disable_ddl_transaction).to be_truthy
   end
 
+  describe "config" do
+    before(:each) do
+      PgHaMigrations.instance_variable_set(:@config, nil)
```

Comment:
Because this runs only as a `before`, it's possible for lines 21-25 to pollute future tests.

The general RSpec pattern for this is to duplicate this into both a `before(:all)` (which handles the first test example that runs) and a `after(:each)` (which ensures every subsequent test is safe). Given that this is just an ivar that will default to not being set anyway, I think just `after(:each)` would be sufficient.